### PR TITLE
Add /openstack/support page

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Added new page at /openstack/support, based on the [copy doc](https://docs.google.com/document/d/1mHaZQ_VZnISTUEpYKiVK-Ejr5ne8lVevpxEc0LpXYKI/edit#)
- Removed an old redirect that sent users from /openstack/support to /advantage
- Added "Support" to the mega nav and the footer navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11166.demos.haus/openstack/support
    - Be sure to test on mobile, tablet and desktop screen sizes
- Make sure that the page matches the copy doc and the links work
- Check that the meganav includes links to the new page

## Issue / Card

Fixes [#4686](https://github.com/canonical-web-and-design/web-squad/issues/4686)

## Screenshots
screencapture-0-0-0-0-8001-openstack-support-2022-01-19-12_17_06 (1)

### CODE REVIEW

File: templates/openstack/support.html [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+        <li>The cloud has to meet local compliance regulations.</li>
+        <li>The cloud has to be operated with minimal downtime.</li>
+      </ul>
+
+      <p>Canonical addresses all those challenges, providing enterprise OpenStack support subscription for OpenStack clouds deployed on Ubuntu.</p>
+
+      <a href="/openstack/pricing-calculator">Get cost estimates for OpenStack support&nbsp;&rsaquo;</a>
+    </div>
+
+    <div class="col-4 u-hide--medium u-hide--small">
```
Comment (petesfrench) [inline templates/openstack/support.html:127]:
```suggestion
    <div class="col-4 u-hide--medium u-hide--small u-vertically-center">
```

File: templates/openstack/support.html [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+  <div class="row">
+    <div class="col-8">
+      <h2>OpenStack support at the best server/annum cost</h2>
+
+      <p>Canonical&#39;s OpenStack support under <a href="/advantage">Ubuntu Advantage</a> subscriptions ensures the best cost efficiency. In 2021, KAUST chose Canonical as a vendor because of the best server/annum cost. Furthermore, with Canonical handling implementation and operational issues, the university has seen quicker problem resolution and significant savings from a manpower perspective. </p>
+
+      <a href="/engage/openstack-cloud-kaust">Read the case study&nbsp;&rsaquo;</a>
+    </div>
+
+    <div class="col-4 u-hide--medium u-hide--small">
```
Comment (petesfrench) [inline templates/openstack/support.html:151]:
```suggestion
    <div class="col-4 u-hide--medium u-hide--small u-vertically-center">
```

**Comment by davegoddard42** [comment]
LGTM @sowasred2012

File: templates/openstack/support.html [hunk 3]
```diff
@@ -0,0 +1,530 @@
...
+    <ol class="p-stepped-list">
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">Design</h3>
+        <p class="p-stepped-list__content">
+          <div class="row u-no-padding">
+            <div class="col-8">
+              <p>Together, we design the cloud according to your needs. We guide you through the hardware specification process to maximise price-performance and tailor cloud architecture to meet your capacity, workload, security, regulatory and integration requirements.</p>
+            </div>
+            <div class="col-4 u-hide--medium u-hide--small">
+              {{ image (
```
Comment (petesfrench) [inline templates/openstack/support.html:321]: Looks like there is a formatting issue here

File: templates/openstack/support.html [hunk 4]
```diff
@@ -0,0 +1,530 @@
...
+          </div>
+        </p>
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">Model</h3>
+        <p class="p-stepped-list__content">
+          <div class="row u-no-padding">
+            <div class="col-8">
+              <p>Once the design step is completed, we translate all design decisions into a model that is reusable across various environments. Modeling guarantees repeatable deployments and enables integration with infrastructure-as-code (IaC) and continuous integration continuous delivery (CICD) systems.</p>
```
Comment (petesfrench) [inline templates/openstack/support.html:340]:
```suggestion
              <p>Once the design step is completed, we translate all design decisions into a model that is reusable across various environments. Modelling guarantees repeatable deployments and enables integration with infrastructure-as-code (IaC) and continuous integration continuous delivery (CICD) systems.</p>
```
British spelling

File: templates/openstack/support.html [hunk 5]
```diff
@@ -0,0 +1,530 @@
...
+      </li>
+    </ol>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Ubuntu Advantage pricing</h2>
+  </div>
+  {% with no_desktop="true" %}{% include "shared/pricing/_ua-for-infrastructure.html" %}{% endwith %}
```
Comment (petesfrench) [inline templates/openstack/support.html:431]: it looks like there is a small difference between this table and the doc, in the row 'Phone and ticket support' under 'Essential', it just has a dash

File: templates/openstack/support.html [hunk 6]
```diff
@@ -0,0 +1,530 @@
...
+  <div class="u-fixed-width">
+    <h1>OpenStack support</h1>
+    
+    <p class="p-heading--4">Use OpenStack with confidence with Ubuntu Advantage for Infrastructure</p>
+    
+    <p>One package, multiple benefits. The most comprehensive Linux enterprise subscription, covering all aspects of open infrastructure.</p>
+    
+    <p>Subscribe to Ubuntu Advantage for Infrastructure and benefit from extended security, certified compliance and 24x7 support.</p>
+
+    <a href="/openstack/support#get-in-touch" class="p-button--positive js-invoke-modal">Get OpenStack support</a>
```
Comment (petesfrench) [inline templates/openstack/support.html:20]: could split these off into their own row>col pattern so they can stack on top of one another on small screens

File: templates/openstack/support.html [hunk 7]
```diff
@@ -0,0 +1,530 @@
...
+
+    <a href="/engage/redhat-openstack-comparison-whitepaper">Read a Red Hat OpenStack comparison whitepaper&nbsp;&rsaquo;</a>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
```
Comment (petesfrench) [inline templates/openstack/support.html:27]: Should use the p-logo-section vanilla pattern here https://vanillaframework.io/docs/patterns/logo-section

File: templates/openstack/support.html [hunk 8]
```diff
@@ -0,0 +1,530 @@
...
+    <ul class="p-inline-images">
+      <li class="p-inline-images__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/7538babd-Bloomberg_logo.svg",
+          alt="Bloomberg",
+          width="130",
+          height="26",
+          hi_def=True,
+          attrs={"class": "p-inline-images__logo"},
+          loading="auto"
```
Comment (petesfrench) [inline templates/openstack/support.html:38]:
```suggestion
          loading="lazy"
```
images that aren't at the top of the page should be set to loading='lazy'

File: templates/openstack/support.html [hunk 9]
```diff
@@ -0,0 +1,530 @@
...
+    </div>
+
+    <div class="col-4 u-hide--medium u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/0853cdb4-kaust.jpeg",
+        alt="",
+        width="200",
+        height="200",
+        hi_def=True,
+        loading="auto|lazy"
```
Comment (petesfrench) [inline templates/openstack/support.html:158]:
```suggestion
        loading="lazy"
```

Comment (petesfrench) [reply templates/openstack/support.html:158]: There are a few others, I haven't highlighted all of them

File: templates/openstack/support.html [hunk 10]
```diff
@@ -0,0 +1,530 @@
...
+  </div>
+</section>
+
+{% with first_item="_openstack_managed", second_item="_cloud_newsletter", third_item="_cloud_engage_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
+
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/openstack" data-form-id="1251" data-lp-id="2086" data-return-url="https://ubuntu.com/openstack/thank-you?product=foundation-cloud" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
+</div>
+
+{% endblock %}
```
Comment (petesfrench) [inline templates/openstack/support.html:530]:
```suggestion
{% endblock %}

```

**Review by petesfrench — COMMENTED** [review]
Some minor changes needed, mostly updating to use the new logo pattern and make sure all but the top image are set to loading="lazy"

File: templates/openstack/support.html (hunk 8 above)
Comment (sowasred2012) [reply templates/openstack/support.html]: These can be seen on large screens, so I've switched them all to auto.

File: templates/openstack/support.html (hunk 7 above)
Comment (sowasred2012) [reply templates/openstack/support.html:27]: Good point!

File: templates/openstack/support.html (hunk 5 above)
Comment (sowasred2012) [reply templates/openstack/support.html:435]: We're using that partial in a few places, and the meaning is the same, so I'll leave that as is.

**Review by petesfrench — APPROVED** [review]
LGTM :+1: 
Is there difference with the table something we need to address?

**Pull request merged**